src/common/Collision/Management/MMapManager.cpp

```diff
@@ -540,7 +540,7 @@ namespace MMAP
             loadedPhasedTiles[swap].insert(packedXY);
 
             // add new swapped tile
-            if (dtStatusSucceed(navMesh->addTile(ptile->data, ptile->fileHeader.size, 0, 0, &loadedTileRefs[packedXY])))
+            if (dtStatusSucceed(navMesh->addTile(ptile->data, ptile->fileHeader.size, DT_TILE_FREE_DATA, 0, &loadedTileRefs[packedXY])))
```

Comment:
so the phased tile is created in memory every time it's swapped and there's no reference anywhere storing the ?
because to me it looks like https://github.com/Carbenium/TrinityCore/blob/mmap-memleak/src/common/Collision/Management/MMapManager.cpp#L467 is going to delete the phased tile